packages/subnav/partials/CtaLinks/icons/github-logo.js

```diff
@@ -5,6 +5,7 @@ function GithubIcon(props) {
     <svg width={16} height={16} viewBox="0 0 16 16" {...props}>
       <path
         fillRule="evenodd"
+        fill="#000000"
```

Comment:
Added this as it's usually present in icon exports, and makes it possible to target the icon's fill in a less brittle way.